### PR TITLE
 Add actual Operator/OCP version in the template 

### PR DESCRIPTION
**Pull request by kpouget**

### CODE REVIEW

File: pkg/artifacts/fetch.go [hunk 1]
```diff
@@ -22,6 +25,7 @@ const (
 	TypeJsonArray ArtifactType = "type:json-array"
 	TypeHtml                   = "type:html"
 	TypeBytes                  = "type:bytes"
+	MissingPageMD5Sum          = "b66c9aae6e6cf88de034b25232ba0181"
```
Comment (jmencak) [inline pkg/artifacts/fetch.go:33]: Hope this is not some md5 hash of some rude word :)  BTW, Isn't MD5 dead?

Comment (kpouget) [reply pkg/artifacts/fetch.go:33]: > Hope this is not some md5 hash of some rude word :) 

not exactly, but the way to find out this hard is still a bit dirty ;-)

I don't know why Google hosting doesn't return a simple 404 code when a page doesn't exist, eg:
https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/logs/periodic-ci-openshift-psap-ci-artifacts-release-4.9-gpu-operator-e2e-master/1437189436143046656/artifacts/gpu-operator-e2e-master/nightly/artifacts/nothing

but instead shows an empty directory :/ so I have to remove all the references to the link I'm trying to access, print the md5sum, and inject it in this constant to detect a missing file :/ :/ :/

>? BTW, Isn't MD5 dead?

is it? maybe when security is involved, not only consistency?

Comment (jmencak) [reply pkg/artifacts/fetch.go:33]: Thank you for the explanation, Kevin.  I believe that a comment in the code (similar to what you said above) would help people in the future trying to make sense out of this.

MD5 is probably fine for checking "404" in this case, but I believe we should stop basically stop using it unless you have a compelling reason for using it.

Comment (kpouget) [reply pkg/artifacts/fetch.go:33]: ack for the md5sum, I'll take it into account next time I need a content validation :)

code fixed with this comment:
```
	// Google GCP hosting doesn't return a 404 error when we request a
	// file that doesn't exist, but instead serves "an empty dir"
	// page.  From this page, we strip all the references to the path
	// we're requesting and compare it against this MD5sum (which was
	// printf-ed and reinjected here :#)
	MissingPageMD5Sum          = "b66c9aae6e6cf88de034b25232ba0181"
```

**Pull request merged**